app/scripts/controllers/channelMonitoring.js

```diff
@@ -267,28 +135,35 @@ angular.module('openhimConsoleApp')
       Alerting.AlertAddMsg('status', 'danger', 'Transaction Status Error: ' + err.status + ' ' + err.data);
     };
 
-    $scope.getStatusMetrics = function(){
+    function updateStatusMetrics() {
+      Api.MetricsChannels.query({
+        channelId : $routeParams.channelId,
+        startDate: moment($scope.selectedDateType.from).format(),
+        endDate: moment($scope.selectedDateType.until).format()
+      }, statusMetricsSuccess, statusMetricsError);
+    }
 
-      // reset any load metric alert warnings
-      Alerting.AlertReset('status');
 
-      var startDate = moment().subtract(6, 'd').toDate();
-      var endDate = moment().toDate();
 
-      Api.Metrics.query({
-        type: 'status',
-        channelId : $routeParams.channelId,
-        startDate: startDate,
-        endDate: endDate
-      }, statusMetricsSuccess, statusMetricsError);
+    $scope.updateMetrics = function() {
+      Alerting.AlertReset('load');
+      Alerting.AlertReset('responseTime');
+      Alerting.AlertReset('status');
 
+      Metrics.refreshDatesForSelectedPeriod($scope.selectedDateType);
+      updateTimeseriesMetrics();
+      updateStatusMetrics();
```

Comment:
Minor: Maybe call this channel metrics. We don't have the concept of status metrics anymore.